<?xml version="1.0" ?>
<launch>
    <arg name="robot0_name" default="mur620a"/>
    <arg name="robot1_name" default="mur620b"/>
    <arg name="robot2_name" default="mur620c"/>
    <arg name="robot3_name" default="mur620d"/>

    <node name="map_reader" pkg="pmadmu_planner" type="map_reader.py" output="screen" />
    <node name="transformation_server" pkg="pmadmu_planner" type="transformation_server.py" output="screen" />
    <node name="visualization" pkg="pmadmu_planner" type="visualization.py" output="screen"/>

    <node name="costmap_reader" pkg="pmadmu_planner" type="costmap_reader.py" output="screen">
        <param name="robot0_name" value="$(arg robot0_name)"/>
        <param name="robot1_name" value="$(arg robot1_name)"/>
        <param name="robot2_name" value="$(arg robot2_name)"/>
        <param name="robot3_name" value="$(arg robot3_name)"/>
    </node>
</launch>
